plugins/hls-hlint-plugin/hls-hlint-plugin.cabal

```diff
@@ -29,6 +29,13 @@ flag pedantic
   default:     False
   manual:      True
 
+flag ghc-lib
+  description:
+    Use ghc-lib-parser rather than the ghc library (requires hlint and
+    ghc-lib-parser-ex to also be built with it)
+  default: True
+  manual: False
```

Comment:
Surely `manual: True`? We only want this to be toggled deliberately, I think.

Reply:
Yes I think you're right.